config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -27,23 +29,46 @@ presubmits:
         resources:
           requests:
             cpu: 4
+  # this replaces the bootstrap / scenario based job going forward
+  - name: pull-kubernetes-integration
+    always_run: true
+    decorate: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
     annotations:
+      fork-per-release: "true"
       testgrid-num-failures-to-alert: '10'
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
-  # this replaces the bootstrap / scenario based job going forward
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191106-09649c7-master
+        command:
+        - runner.sh
+        args:
+        - ./hack/jenkins/test-dockerized.sh
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 4
   - name: pull-kubernetes-integration
     always_run: true
     decorate: true
     branches:
-    - master
+    - release-1.17 # per-release job
```

Comment:
so should this job go in `1.17.yaml`?

Reply:
i'm not sure. IMO it appears those are all the auto generated forked jobs and this was done by hand, i'm hesitant to re-fork the jobs, from what I can tell that's meant to be done point in time.

it should _work_ here.

Reply:
that's fine. I just wasn't sure what was normal